Gemfile.lock.release

```diff
@@ -913,7 +908,7 @@ GEM
     manageiq-loggers (1.2.1)
       activesupport (>= 5.0)
       manageiq-password (< 2)
-    manageiq-messaging (1.5.0)
+    manageiq-messaging (2.0.0)
```

Comment:
NOTE this breaking change in manageiq-messaging was to not default protocol to STOMP which is fine since we never set that up with appliance_console anyway